packages/config/config/devices/0x027a/zen31.json

```
@@ -274,11 +157,11 @@
 		{
 			"#": "152",
 			"label": "Ramp Rate (Z-Wave)",
-			"description": "How long it takes for the dimmer to reach full brightness or turn off when operated remotely from the Z-Wave hub.",
+			"description": "1-127 = seconds, 128-254 = 1-127 minutes",
```

For consistency with other configurations I prefer
```suggestion
			"description": "1...127 = seconds, 128...254 = minutes (minus 127)",
```